test/fixtures/discordResponse/discord-response.js

```diff
@@ -68,6 +68,52 @@ const getDiscordMembers = [
     mute: false,
     deaf: false,
   },
+  {
+    avatar: "",
+    communication_disabled_until: "",
+    flags: 0,
+    is_pending: false,
+    joined_at: "some_intensional_date", // ISO8601 timestamp
+    nick: "goliath",
+    pending: false,
+    premium_since: null,
+    roles: ["giant", "hero"],
```

Comment:
move required role to this test data collection

Reply:
I had to add roles separately in order to avoid messing with other existing tests